wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/InternalMessages.java

```diff
@@ -119,6 +119,16 @@
 	 * The key for the invalid pattern error text.
 	 */
 	public static final String DEFAULT_VALIDATION_ERROR_INVALID_PATTERN = "bordertech.wcomponents.message.fieldInvalidPattern";
+	
+	/**
+	 * The key for invalid file type.
+	 */
+	public static final String DEFAULT_VALIDATION_ERROR_FILE_WRONG_TYPE = "bordertech.wcomponents.message.fileWrongType";
+
+	/**
+	 * The key for invalid file size.
+	 */
+	public static final String DEFAULT_VALIDATION_ERROR_FILE_WRONG_SIZE = "bordertech.wcomponents.message.fileTooLarge";
```

Comment:
Might be better to match the parameter name and variable name (same as the file wrong type), just for consistency